libethereum/State.cpp

```diff
@@ -541,6 +541,7 @@ std::pair<ExecutionResult, TransactionReceipt> State::execute(EnvInfo const& _en
 	u256 const startGasUsed = _envInfo.gasUsed();
 	executeTransaction(e, _t, onOp);
 
+	res.changeLog = m_changeLog;
```

Comment:
Is the "state diff" the same as "state journal" or "state changelog"? If yes, let's stick to a single name. We can add a function to export the changelog next to the functions like `savepoint()` or `revert()`.